src/Bundle/DependencyInjection/Configuration.php

```diff
@@ -39,6 +39,9 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->arrayNode('mapping')
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->arrayNode('imports')
```

Comment:
This new path ensures we do not load any files that are on Autoloader which cannot be include serveral times.